setup.py

```diff
@@ -44,19 +70,30 @@ def get_version(module='spyder_kernels'):
 setup(
     name='spyder-kernels',
     version=get_version(),
-    keywords='spyder jupyter kernel',
+    keywords='spyder jupyter kernel ipython console shell',
     url='https://github.com/spyder-ide/spyder-kernels',
+    download_url="https://www.spyder-ide.org/#fh5co-download",
     license='MIT',
     author='Spyder Development Team',
-    description='Jupyter kernels for the Spyder console',
+    author_email="spyderlib@googlegroups.com",
+    description="Jupyter kernels for the Spyder IDE's IPython consoles.",
```

Comment:
The previous description is fine:

1. We only have one console, so there's no need for `IPython console`.
2. IDE is ugly. You don't see the Pycharm IDE, just Pycharm.

So, this should be:

    Jupyter kernels for the Spyder's console

Reply:
Both good points; I include "IDE" because (unlike PyCharm) not everyone knows what Spyder is, but I agree it looks ugly. Reverted to (fixed grammar):

```
Jupyter kernels for Spyder's console
```